mappings/net/minecraft/client/render/entity/model/VexEntityModel.mapping

```diff
@@ -8,12 +8,6 @@ CLASS net/minecraft/class_617 net/minecraft/client/render/entity/model/VexEntity
 	COMMENT   <th>Part Name</th><th>Parent</th><th>Corresponding Field</th>
 	COMMENT </tr>
 	COMMENT <tr>
-	COMMENT   <td>{@value EntityModelPartNames#HAT}</td><td>Root part</td><td>{@link #hat}</td>
-	COMMENT </tr>
-	COMMENT <tr>
-	COMMENT   <td>{@value EntityModelPartNames#HEAD}</td><td>Root part</td><td>{@link #head}</td>
```

Comment:
Vex still has a head in its model

Reply:
Hmm? There is no field for it?

Reply:
No field, but it's added to the ModelPartData in getTexturedModelData()

Reply:
Hmm, how do we document those?